source/seedsource/management/commands/populate_services.py

```diff
@@ -0,0 +1,122 @@
+WGS84 = '+proj=latlong +datum=WGS84 +no_defs'
+
+
+class Command(BaseCommand):
+    help = 'Populates a region\'s services with a DEM and ClimateNA clipped to the region.'
+
+    def add_arguments(self, parser):
+        parser.add_argument('region_name', nargs=1, type=str)
+
+    def handle(self, region_name, *args, **options):
+        name = region_name[0]
+
+        from django.conf import settings
+        BASE_DIR = settings.NC_SERVICE_DATA_ROOT
+
+        # determine extent and lat/lon variable names from DEM
+        dem_path = os.path.join(BASE_DIR, 'regions', name, '{}_dem.nc'.format(name))
+        with Dataset(dem_path, 'r') as ds:
+            dims = ds.dimensions.keys()
+            lat = 'lat' if 'lat' in dims else 'latitude'
+            lon = 'lon' if 'lon' in dims else 'longitude'
+            l = float(ds.variables[lon][:].min())
+            b = float(ds.variables[lat][:].min())
+            r = float(ds.variables[lon][:].max())
+            t = float(ds.variables[lat][:].max())
+            extent = BBox((l, b, r, t), projection=pyproj.Proj(WGS84))
+
+        # Generate DEM service
+        with transaction.atomic():
+            print('Adding {}'.format(name))
+            print('---')
+            print('elevation')
+            service_name = '{}_dem'.format(name)
+            if not Service.objects.filter(name__iexact=service_name).exists():
+
+                dem_service = Service.objects.create(
+                    name=service_name,
+                    data_path='regions/{name}/{name}_dem.nc'.format(name=name),
+                    projection=WGS84,
+                    full_extent=extent,
+                    initial_extent=extent
+                )
+                with Dataset(dem_path, 'r') as ds:
+                    v_min = numpy.nanmin(ds.variables['elevation'][:]).item()
+                    v_max = numpy.nanmax(ds.variables['elevation'][:]).item()
+                    renderer = StretchedRenderer([(v_min, Color(0, 0, 0)), (v_max, Color(255, 255, 255))])
+                    variable = Variable.objects.create(
+                        service=dem_service,
+                        index=0,
+                        variable='elevation',
+                        projection=WGS84,
+                        x_dimension=lon,
+                        y_dimension=lat,
+                        name='elevation',
+                        renderer=renderer,
+                        full_extent=extent
+                    )
+            else:
+                print('{} already exists, skipping.'.format(service_name))
+
+        # Generate ClimateNA services
+        with transaction.atomic():
+            for year in YEARS:
+
+                # Bandaid fix for missing 1981_2010 data for ne1, remove when available
+                if name == 'ne1' and year == '1981_2010':
+                    continue
+
+                print(year)
```

Comment:
Print a blank line before each year for formatting. So instead of:

```
Adding nc1
---
elevation
1961_1990
---
MAT
...
```

you'd get:

```
Adding nc1
---
elevation

1961_1990
---
MAT
```